build_system/seqan3-config-version.cmake

```diff
@@ -39,6 +39,18 @@ else()
     endif()
 endif()
 
+# extract release candidate
+file(STRINGS "${SEQAN3_INCLUDE_DIR}/seqan3/version.hpp" SEQAN3_RELEASE_CANDIDATE_HPP REGEX "#define SEQAN3_RELEASE_CANDIDATE ")
+string(REGEX REPLACE "#define SEQAN3_RELEASE_CANDIDATE " "" SEQAN3_RELEASE_CANDIDATE_VERSION "${SEQAN3_RELEASE_CANDIDATE_HPP}")
+
+if (SEQAN3_RELEASE_CANDIDATE_VERSION VERSION_GREATER "0")
+    set (SEQAN3_VERSION "${PACKAGE_VERSION}-rc.${SEQAN3_RELEASE_CANDIDATE_VERSION}")
+else ()
+    set (SEQAN3_VERSION "${PACKAGE_VERSION}")
```

Comment:
If we do not run `build_system/seqan3-config.cmake`, we still need the SEQAN3_VERSION for documentation builds